examples/fungible-token/src/lib.rs

```diff
@@ -16,263 +16,65 @@
-    /// * If this function is called by an escrow account (`owner_id != predecessor_account_id`),
-    ///   then the allowance of the caller of the function (`predecessor_account_id`) on
-    ///   the account of `owner_id` should be greater or equal than the transfer `amount`.
-    /// * Caller of the method has to attach deposit enough to cover storage difference at the
-    ///   fixed storage price defined in the contract.
-    #[payable]
-    pub fn transfer_from(&mut self, owner_id: AccountId, new_owner_id: AccountId, amount: U128) {
-        let initial_storage = env::storage_usage();
-        assert!(
-            env::is_valid_account_id(new_owner_id.as_bytes()),
-            "New owner's account ID is invalid"
-        );
-        let amount = amount.into();
-        if amount == 0 {
-            env::panic(b"Can't transfer 0 tokens");
-        }
-        assert_ne!(
-            owner_id, new_owner_id,
-            "The new owner should be different from the current owner"
-        );
-        // Retrieving the account from the state.
-        let mut account = self.get_account(&owner_id);
-
-        // Checking and updating unlocked balance
-        if account.balance < amount {
-            env::panic(b"Not enough balance");
-        }
-        account.balance -= amount;
-
-        // If transferring by escrow, need to check and update allowance.
-        let escrow_account_id = env::predecessor_account_id();
-        if escrow_account_id != owner_id {
-            let allowance = account.get_allowance(&escrow_account_id);
-            if allowance < amount {
-                env::panic(b"Not enough allowance");
-            }
-            account.set_allowance(&escrow_account_id, allowance - amount);
-        }
-
-        // Saving the account back to the state.
-        self.set_account(&owner_id, &account);
-
-        // Deposit amount to the new owner and save the new account to the state.
-        let mut new_account = self.get_account(&new_owner_id);
-        new_account.balance += amount;
-        self.set_account(&new_owner_id, &new_account);
-        self.refund_storage(initial_storage);
+        Self { token: near_std::token::Token::new(owner_id, total_supply.into()) }
     }
+}
 
-    /// Transfer `amount` of tokens from the caller of the contract (`predecessor_id`) to
-    /// `new_owner_id`.
-    /// Act the same was as `transfer_from` with `owner_id` equal to the caller of the contract
-    /// (`predecessor_id`).
-    /// Requirements:
-    /// * Caller of the method has to attach deposit enough to cover storage difference at the
-    ///   fixed storage price defined in the contract.
-    #[payable]
-    pub fn transfer(&mut self, new_owner_id: AccountId, amount: U128) {
-        // NOTE: New owner's Account ID checked in transfer_from.
-        // Storage fees are also refunded in transfer_from.
-        self.transfer_from(env::predecessor_account_id(), new_owner_id, amount);
+// #[near_bindgen]
```

Comment:
How does it work without this attribute?

Reply:
It doesn't work with or without actually. Filed an issue in #276